infrastructures/infrastructure-common/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/ConnectorIaasController.java

```diff
@@ -314,18 +315,22 @@ public void executeScriptWithKeyAuthentication(String infrastructureId, String i
     private void runScriptOnInstance(String infrastructureId, String instanceId, String instanceScriptJson)
             throws ScriptNotExecutedException {
         String scriptResult = null;
+        String scriptsInJson = null;
```

Comment:
why scripts in plural ?

Reply:
Not a mistake: The structure is an (JSON) array containing multiple scripts.